src/assets/i18n/en_US.json

```diff
@@ -297,20 +298,20 @@
   "account.organization.cost_center_management.create.confirmation": "The cost center \"{{0}}\" has been created.",
   "account.organization.cost_center_management.delete.confirmation": "The cost center has been deleted.",
   "account.organization.cost_center_management.update.confirmation": "The cost center \"{{0}}\" has been updated.",
-  "account.organization.hierarchies": "Company Structure",
+  "account.organization.hierarchies": "Company structure",
   "account.organization.hierarchies.grouppath.tooltip.close": "Collapse",
-  "account.organization.hierarchies.grouppath.tooltip.open": "Show All Groups",
-  "account.organization.hierarchies.groups.button.add.label": "Add Group",
-  "account.organization.hierarchies.groups.delete.confirmation": "The group with id \"{{0}}\" has been deleted.",
+  "account.organization.hierarchies.grouppath.tooltip.open": "Show all groups",
+  "account.organization.hierarchies.groups.button.add.label": "Add group",
+  "account.organization.hierarchies.groups.delete.confirmation": "The group with ID \"{{0}}\" has been deleted.",
   "account.organization.hierarchies.groups.new.button.label": "Create",
   "account.organization.hierarchies.groups.new.confirmation": "The group \"{{0}}\" has been created.",
-  "account.organization.hierarchies.groups.new.message": "Creating a new group to track your company structure. You can assign users to it.",
+  "account.organization.hierarchies.groups.new.message": "Creating a new group to reflect your company structure. You can assign users to it.",
```

Comment:
Maybe the gerund "Creating ..." is not the best solution here. In German this is translated as a call for action while the French translation sounds more like a title than a sentence. 
I'd suggest changing the English source and adjust the localizations accordingly.

Reply:
Create instead of creating.